src/System.Security.Cryptography.X509Certificates/tests/System.Security.Cryptography.X509Certificates.Tests.csproj

```diff
@@ -4,37 +4,33 @@
   <PropertyGroup>
     <ProjectGuid>{A28B0064-EFB2-4B77-B97C-DECF5DAB074E}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <HaveDSA Condition="'$(TargetGroup)'=='netcoreapp' OR '$(TargetGroup)'=='uap' OR '$(TargetGroup)'=='net462'">true</HaveDSA>
```

Comment:
TargetGroup will never be net462 nor net472. TargetGroup can only be one of the ones in the Configurations.props file of the project. Basically the way it works is that you are building a specific vertical (for example, netfx) and then we will decide which TargetGroup applies best for this specific project, and that will be the value of TargetGroup. For that reason, I would simply just leave all of these conditions as `'$(TargetGroup)' != 'netstandard'`. If you really want to cross compile for net472 or net462, then you have to add those configs into the Configurations.props file